microproxy/log.py

```diff
@@ -49,5 +51,6 @@ def register_stream_handler(cls):  # pragma: no cover
 
     @classmethod
     def get_logger(cls, name):  # pragma: no cover
-        logger = logging.getLogger(name)
+        shortname = ".".join(split_str for split_str in name.split(".") if split_str != "microproxy")
+        logger = logging.getLogger(shortname)
```

Comment:
Is this code mean remove microproxy for the logger name?
Maybe it need a comment or extract a function for it.
It's not clear to see the code and know what this code mean.

BTW, do we really need to remove microproxy prefix of logger name?
What could we get the benefit from it?